app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+  def current_user
+    session[:email] = "ci@example.com" if bypass_auth?
+    @current_user ||= session[:email]
+  end
+
+  def user_signed_in?
+    !!current_user
+  end
+
+  def bypass_auth?
+    Rails.configuration.x.bypass_auth == true
+  end
```

Comment:
[suggestion (nonblocking)] these all feel more like controller methods to me than helper methods, I'd move them up to `ApplicationController` and then expose the one(s) you need in views with `helper_method :method_name`